.gitignore

```diff
@@ -39,3 +39,6 @@ docs/generated
 
 # Default pycharm virtual env
 .venv/
+
+# Notebook formatting script.
+nbformat
```

Comment:
but it downloads into `dev_tools/`, right? I guess this still works but may be overzealous. Probably doesn't matter too much

Reply:
Indeed downloads into `dev_tools`